source/kernel/kernel.go

```diff
@@ -97,6 +101,21 @@ func parseVersion() (map[string]string, error) {
 	}
 
 	full := strings.TrimSpace(string(raw))
+
+	// Replace forbidden symbols for Kubernetes tags
+	for _, symbol := range Config.KernelVersionReplace {
+		full = strings.Replace(full, symbol, "_", -1)
+	}
+
+	// Check if string matches standard
+	matched, err := regexp.MatchString(`^[[a-zA-Z0-9]+[a-zA-Z0-9-_.]*[a-zA-Z0-9]+$`, full)
+	if err != nil {
+		log.Printf("ERROR: Failed to read regex for : %s", full)
+	}
+	if !matched {
+		log.Printf("ERROR: Kernel version full should consist of alphanumeric characters, '-', '_' or '.', and must start and end with an alphanumeric character : %s", full)
+	}
```

Comment:
I think we can make this much simpler. Just replace all forbidden characters without any configurability.

E.g. global
```
re := regexp.MustCompile("[^-A-Za-z0-9_.]")
```

And the for the full version label:
```
fmt.Println(re.ReplaceAllString(value, "_"))
```